reference/examples/LazyCreateAccount.md

```diff
@@ -0,0 +1,18 @@
+Lazy-create a new account using a public-address via the CryptoCreate transaction.
+
+Reference: HIP-583
+
+## Example 1
+- Create an Ethereum account address/public-address (example: 0xb794f5ea0ba39494ce839613fffba74279579268)
+- Use the `AccountCreateTransaction` to create an account using the Ethereum account address/public address
+  - Use the `setAliasEVMAddress()` 
+  - Do not use the `setKey` field or `setAlias()` field
+- Sign with... // what key? The testnet account ID and key or the private key of the Ethereum account public-address?
+- Execute the transaction
+- Show the transaction was successful by getting the transaction status from the receipt and new Hedera Account ID
+- Show the account that was created does not have an account public key defined
+- Use the public-address to transfer hbars to that address in a `TransferTransaction`
+  - Use the account that was created in the previous step as transaction fee payer for the `TransferTransaction`
+     - This will assign a public key to the account that is not present when the account is created from the transaction fee paying account
```

Comment:
What account/key/address is signing the transfer transaction so that the public key can be extracted and applied?

Reply:
The account who owns the public key that maps to the evm address